aim/sdk/objects/image.py

```diff
@@ -136,27 +131,51 @@ def _from_pil_image(self, pil_image: PILImage.Image, params):
         assert isinstance(pil_image, PILImage.Image)
         img_container = BytesIO()
 
-        try:
-            pil_image.save(img_container, **params)
-        except OSError as exc:
-            # The best way to approach this problem is to prepare PIL Image object before hitting this method.
-            # This block only handles case where RGBA/P/LA/PA mode is mandated to save in RGB
-            # PIL won't do that automatically, so we have to convert image to RGB before saving it.
-            # In addition - make transparency "white" before conversion otherwise it will be black.
-            if pil_image.mode not in ('RGBA', 'LA', 'PA', 'P'):
-                raise
-            elif not Image.FLAG_WARN_RGBA_RGB:
-                logger.warning(f'Failed to save the image due to the following error: {exc}')
-                logger.warning(f'Attempting to convert mode "{pil_image.mode}" to "RGB"')
-                Image.FLAG_WARN_RGBA_RGB = True
-
-            alpha = pil_image.convert('RGBA').split()[-1]  # Get only alpha
-            background = PILImage.new('RGBA', pil_image.size, (255, 255, 255, 255))
-            background.paste(pil_image, mask=alpha)
-            pil_image = background.convert('RGB')
-
-            # Retry
-            pil_image.save(img_container, **params)
+        if not params['format']:
+            params['format'] = pil_image.format or 'PNG'
```

Comment:
why switch to uppercase? the 'format' is [mostly] used internally, what's the motivation of this change?

Reply:
I wanted to alight with PIL standard but since its only for internal I'll revert it